python/tests/test_programmatically_create_an_AIConfig.py

```diff
@@ -577,17 +577,62 @@ def test_add_output_existing_prompt_no_overwrite(ai_config_runtime: AIConfigRunt
     test_result2 = ExecuteResult(
         output_type="execute_result",
         execution_count=0.0,
-        data={"role": "assistant", "content": "test output"},
+        data={"role": "assistant", "content": "test output for second time"},
         metadata={"finish_reason": "stop"},
     )
     ai_config_runtime.add_output("GreetingPrompt", test_result2)
     assert ai_config_runtime.get_latest_output("GreetingPrompt") == test_result2
 
     ai_config_runtime.delete_output("GreetingPrompt")
-
     assert ai_config_runtime.get_latest_output("GreetingPrompt") == None
 
+def test_add_output_existing_prompt_overwrite(ai_config_runtime: AIConfigRuntime):
+    """Test adding an output to an existing prompt with overwriting."""
+    base_result = ExecuteResult(
+        output_type="execute_result",
+        execution_count=0.0,
+        data={"role": "assistant", "content": "base output"},
+        metadata={"finish_reason": "stop"},)
+    prompt1 = Prompt(
+        name="GreetingPrompt",
```

Comment:
@rossdanlm I this what you were suggesting?

Reply:
This is better thanks, but let's also change:

1. rawResponse --> `raw_response`
2. data --> make it string format: `data="original_output"`

Reply:
Done! Thanks for pointing this out. I kinda overlooked the details. 😿